src/main/kotlin/skytils/skytilsmod/features/impl/misc/ItemFeatures.kt

```diff
@@ -86,9 +86,9 @@ class ItemFeatures {
                     if (event.slot.hasStack) {
                         val stack = event.slot.stack
                         if (stack.displayName.contains("Health Potion")) event.slot highlight Color(255, 225, 30, 255)
-                        else if (stack.displayName.contains("Mimic Fragment") || stack.displayName.contains("Training Weights") || stack.displayName.contains(
-                                "Journal Entry"
-                            ) || StringUtils.containsAny(stack.displayName, "Defuse Kit", "Lever", "Torch", "Stone", "Button", "Tripwire Hook")
+                        else if (StringUtils.containsAny(stack.displayName, "Defuse Kit", "Lever", "Torch", 
+                                                         "Stone", "Button", "Tripwire Hook", "Journal Entry", 
```

Comment:
why does it highlight stone too it's supposed to be stone buttons

Reply:
good point